### PR TITLE
[ci] Fix ffmpeg Url

### PR DESCRIPTION
**Pull request by naveen521kk**
## List of Changes
- Fix Ffmpeg URL for windows ci as the previous upstream is down for a week.

## Motivation
TO prevent failing tests for windows which could happen when Github Cache expires next month.

## Acknowledgement
- [x] I have read the [Contributing Guidelines](https://github.com/ManimCommunity/manim/wiki/Documentation-guidelines-(WIP))

### CODE REVIEW

File: .github/manimdependency.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "windows": {
         "sox": "sox-14.4.2-win32",
-        "ffmpeg": "ffmpeg-4.3.1-win64-static"
+        "ffmpeg": "https://github.com/BtbN/FFmpeg-Builds/releases/download/autobuild-2020-09-24-12-32/ffmpeg-N-99357-g14d6838638-win64-gpl.zip"
```
Comment (behackl) [inline .github/manimdependency.json:4]: I don't particularly like that we are switching from `ffmpeg-4.3.1` to a nightly build. What about the released version that can be found at https://www.gyan.dev/ffmpeg/builds/ ?

Comment (naveen521kk) [reply .github/manimdependency.json]: Just because it was Github. ;-)

**Review by behackl — APPROVED** [review]
Judging from the passing pipeline, this works. :-)

Thanks for fixing this!

**Comment by naveen521kk** [comment]
This needs an review to be merged.

**Review by kilacoda — APPROVED** [review]

**Pull request merged**